src/utils.cc

```diff
@@ -740,14 +757,31 @@ bool utils::anyParameterAtBoundaries( const RooArgSet &params, int verbosity ){
 
     RooLinkedListIter iter = params.iterator(); int i = 0;
     for (RooRealVar *a = (RooRealVar *) iter.Next(); a != 0; a = (RooRealVar *) iter.Next(), ++i) {
-
-        bool isBad = isParameterAtBoundary(*a);
+      
+        bool atLowerBoundary = isParameterAtLowerBoundary(*a);
+        bool atUpperBoundary = isParameterAtLowerBoundary(*a);
```

Comment:
On 06/07/15 04:06, André David wrote:
>
> In src/utils.cc 
> <https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit/pull/197#discussion_r33902452>:
>
> > @@ -740,14 +757,31 @@ bool utils::anyParameterAtBoundaries( const RooArgSet &params, int verbosity ){
> >
> >      RooLinkedListIter iter = params.iterator(); int i = 0;
> >      for (RooRealVar *a = (RooRealVar *) iter.Next(); a != 0; a = (RooRealVar *) iter.Next(), ++i) {
> > -
> > -        bool isBad = isParameterAtBoundary(*a);
> > +
> > +        bool atLowerBoundary = isParameterAtLowerBoundary(*a);
> > +        bool atUpperBoundary = isParameterAtLowerBoundary(*a);
>
> This is bugged: should be atUpperBoundary = isParameterAtUpperBoundary.
>
ah.. thanks for spotting this (it didn't reveal itself in my test case 
where the parameter
is close to both boundaries). I'll update this.